### ISSUE TITLE
add proper listing of remote backups

### ISSUE THREAD

**Issue by barazbay** [Altinity/clickhouse-backup#146]
If we create backup and then upload it to the remote storage, backup name will contain archive prefix

```
# clickhouse-backup list --config=/etc/clickhouse-backup/config-daily.yml 
Local backups:
no backups found
Remote backups:
- '2021-01-29T09-03-21.tar'	30.50 KiB	(created at 29-01-2021 09:03:45)

```
and then, if we will try to download backup '2021-01-29T09-03-21.tar' from the remote storage
we will get an error
```
# clickhouse-backup download '2021-01-29T09-03-21.tar' --config=/etc/clickhouse-backup/config-daily.yml 
2021/01/29 11:47:21 file not found

and directory with tar name will be created
# ls -l /var/lib/clickhouse/backup/2021-01-29T09-03-21.tar/
total 0
```

in order to properly download backup we must omit archive prefix
```
# clickhouse-backup download '2021-01-29T09-03-21' --config=/etc/clickhouse-backup/config-daily.yml 
 30.50 KiB / 30.50 KiB [====================================================================================================================================================================] 100.00% 0s
2021/01/29 11:48:23   Done.

```
Please, update documentation and/or the ouput of list command

**Comment by AlexAkulov** [comment]
Thank you! 
I will fix it in next version.

**Comment by AlexAkulov** [comment]
Fixed in [v1.0.0-alpha1](https://github.com/AlexAkulov/clickhouse-backup/releases/tag/v1.0.0-alpha1)
